lib/service/invoicesubscription.go

```diff
@@ -236,14 +243,16 @@ func (svc *LndhubService) InvoiceUpdateSubscription(ctx context.Context) error {
 	for {
 		select {
 		case <-ctx.Done():
-			return fmt.Errorf("Context was canceled")
+			return context.Canceled
```

Comment:
@kiwiidb this is a differnt/correct type of error, correct?

Reply:
it's just a slightly different string (the standard one from the context lib)